### DISCUSSION TITLE
Development Server HTTP ERROR 426 with WSL2

### DISCUSSION THREAD

**Discussion by grtn91** [strapi/strapi#6020] [Legacy Topics]
**Describe the bug**
Hello, 
currently I'm running a WSL2 Instance of Ubuntu 18.04. I created an empty folder and created a project. When the browser opens the url localhost:1337 i get an Http Error 426.

**Steps to reproduce the behavior**

1. Create new folder
2. Enter folder
3. Run "npx create-strapi-app . --quickstart"
4. Wait until browser opens
5. Browser opens with HTTP Error 426

**Expected behavior**
Development Server would ran and I can create a admin account. 

**Screenshots**
https://ibb.co/pQQdNPr
https://ibb.co/5RFQxJv

**System**

- Node.js version: v13.12.0
- NPM version: 6.14.14
- Strapi version: latest
- Database: sqlite
- Operating system: Windows 10 with WSL2 Ubuntu

**Additional context**

**Comment by derrickmehaffy** [comment]
Unable to reproduce in WSL/Windows 10, this doesn't sound like a Strapi bug but a user environment issue.

**Comment by grtn91** [comment]
@derrickmehaffy  youre using WSL2? I tried going back to the old WSL1 Version, then i got the error:

`[2020-05-01T12:45:21.073Z] error Error: bind EACCES 0.0.0.0:1337`

Which led me to the following issue:
https://github.com/strapi/strapi/issues/5057

So changing the port from 1337 to 3008 in the enviroments config it also works for me now.

**Comment by derrickmehaffy** [comment]
This seems like a problem with WSL2 and bindings (although yes I was testing on WSL2 and could not reproduce, meaning you have something running on port 1337 or something with your WSL2 won't let you bind on certain port ranges.)

For now I'm going to move this to a discussion and mark your reply as the answer, as it's not really a bug of Strapi but that of WSL2/User Environment
